Installments/index.jsx

```diff
@@ -13,6 +13,59 @@ const classes = {
   cellHighlight: `${baseClass}__cell__highlight`
 }
 
+const TARGET_FPS = 60
+
+const requestAnimationFrame = (fn) => {
+  if (window.requestAnimationFrame) {
+    window.requestAnimationFrame(fn)
+  } else {
+    setTimeout(fn, 1000 / TARGET_FPS)
+  }
+}
+
+const debounce = (fn) => {
+  let waiting = false
+
+  return function (...args) {
+    if (!waiting) {
+      waiting = true
+
+      requestAnimationFrame(() => {
+        fn(...args)
+        waiting = false
+      })
```

Comment:
Yeah, I know what you mean, but the problem is not doing the motion, it's setting the state on resize. We need to recalculate the position on resize and store that in the state. This is not the position and dimensions while in transition, but the final position and dimensions after the transition has completed. If we try to do that on every resize, we'll completely kill performance, since we have to layout in order to find that position.